src/CMakeLists.txt

```
@@ -0,0 +1,483 @@
+    V3Inline.cpp                  
+    V3Inst.cpp                    
+    V3InstrCount.cpp              
+    V3Life.cpp                    
+    V3LifePost.cpp                
+    V3LinkCells.cpp               
+    V3LinkDot.cpp                 
+    V3LinkInc.cpp                 
+    V3LinkJump.cpp                
+    V3LinkLevel.cpp               
+    V3LinkLValue.cpp              
+    V3LinkParse.cpp               
+    V3LinkResolve.cpp             
+    V3Localize.cpp                
+    V3MergeCond.cpp               
+    V3Name.cpp                    
+    V3Number.cpp                  
+    V3OptionParser.cpp            
+    V3Options.cpp                 
+    V3Order.cpp                   
+    V3Os.cpp                      
+    V3Param.cpp                   
+    V3Partition.cpp               
+    V3PreShell.cpp                
+    V3Premit.cpp                  
+    V3ProtectLib.cpp              
+    V3Randomize.cpp               
+    V3Reloop.cpp                  
+    V3Sched.cpp                   
+    V3SchedAcyclic.cpp            
+    V3SchedPartition.cpp          
+    V3SchedReplicate.cpp          
+    V3SchedTiming.cpp             
+    V3Scope.cpp                   
+    V3Scoreboard.cpp              
+    V3Slice.cpp                   
+    V3Split.cpp                   
+    V3SplitAs.cpp                 
+    V3SplitVar.cpp                
+    V3Stats.cpp                   
+    V3StatsReport.cpp             
+    V3String.cpp                  
+    V3Subst.cpp                   
+    V3Table.cpp                   
+    V3Task.cpp                    
+    V3Trace.cpp                   
+    V3TraceDecl.cpp               
+    V3Tristate.cpp                
+    V3TSP.cpp                     
+    V3Undriven.cpp                
+    V3Unknown.cpp                 
+    V3Unroll.cpp                  
+    V3VariableOrder.cpp           
+    V3Waiver.cpp                  
+    V3Width.cpp                   
+    V3WidthSel.cpp                
+    V3ParseImp.cpp
+    V3ParseGrammar.cpp
+    V3ParseLex.cpp
+    V3PreProc.cpp
+)
+
+SET(COVERAGE_SOURCES
+    VlcMain.cpp
+)
+
+# Note about tests:
+# VlcMain.cpp #includes the following files:
+#  V3Error.cpp, V3String.cpp, V3Os.cpp and VlcTop.cpp
+# V3Number_test.cpp #includes the following files:
+#  V3FileLine.cpp
+
+#
+# Generated sources and headers for the verilator binary
+
+set(srcdir ${CMAKE_CURRENT_SOURCE_DIR})
+
+file(TO_NATIVE_PATH ${srcdir}/astgen ASTGEN)
+file(TO_NATIVE_PATH ${srcdir}/bisonpre BISONPRE)
+file(TO_NATIVE_PATH ${srcdir}/flexfix FLEXFIX)
+file(TO_NATIVE_PATH ${srcdir}/../bin/verilator_includer VERILATOR_INCLUDER)
+file(TO_NATIVE_PATH ${srcdir}/vlcovgen VLCOVGEN)
+file(TO_NATIVE_PATH ${srcdir}/config_rev CONFIG_REV)
+
+configure_file(config_build.h.in config_build.h @ONLY)
+
+add_custom_command(
+    OUTPUT V3Ast__gen_forward_class_decls.h V3Dfg__gen_forward_class_decls.h
+    DEPENDS ./V3Ast.h ${ASTGEN}
+    COMMAND python ARGS
```

```suggestion
    COMMAND ${PYTHON3} ARGS
```

Or equivalent, sometimes people need to point at a specific version.